cli/src/main.rs

```diff
@@ -1,13 +1,15 @@
 use clap::{Parser, Subcommand};
 
 mod add_url;
-use add_url::append_url_to_db;
+use add_url::append_url;
 
 mod auth;
 use auth::check_auth_status;
 use auth::get_user_id_from_session;
 
 mod sign_up;
+// use db::urls::process_page_snapshot;
+// use db::urls::AddUrlOutcome;
```

Comment:
Nit: Lets remove these!

Don't like leaving commented code on the main branch basically lol